### ISSUE TITLE
Trying to login using Firefox after signing up in Chrome forces perpetual display of "Email Activation Link" error message

### ISSUE THREAD

**Issue by Mike-T-Mitchell** [mitro-co/mitro#118]
After logging in to/creating my Mitro account using Chrome, I cannot login to Mitro using Firefox without seeing the error message "Please click the verification link we sent to your email" with a 7-digit ID appearing after the message. I'm sure I'm using the correct email and (case-sensitive) password, and given that I opened this account almost a year ago, I'm sure the verification link was clicked back when it was opened. I use the app everyday. 

My guess is that this specific error message doesn't actually describe the correct error I SHOULD be seeing. I'm not sure if the number of users logged into the same account is limited or not (as I am logged into the same account in Chrome while trying to login on Firefox, albeit on the same machine), though I'm wondering if that's a possible reason why I keep seeing that message. 

It seems that the reason I'm getting only that specific error message over and over again is because the exception handler only contains that error message -- https://github.com/mitro-co/mitro/blob/master/mitro-core/java/server/src/co/mitro/core/exceptions/DoEmailVerificationException.java

A screenshot of the error I'm getting is attached. 

Thanks in advance for any help/advice offered =)

Have a great one!
![screenshot 2015-05-05 17 22 47](https://cloud.githubusercontent.com/assets/7085327/7483254/639e2d42-f34b-11e4-8930-3458744e292b.png)

**Comment by winstonhong** [comment]
Hello Mike-T-Mitchell.

Mitro does NOT limit the number of devices.

The issue is that Firefox 32 and later versions contain the latest security patches, which break some functions of Mitro.

Would you please try to use Chrome to login your Mitro account?

More details can be refer to Issue #107 and Issue #108.

**Comment by vijayp** [comment]
Yeah, the new ff features do break things pretty badly. Unfortunately, we (Evan, Adam, and I) don't really have much time to maintain the code anymore. We need to figure out a more scalable solution to handle pull requests and manage the entire process. I'm open to any ideas from the community!

**Comment by Mike-T-Mitchell** [comment]
Thank you guys for the reply. I completely understand. Turns out that the issue was this -- I was being sent an e-mail (which was ending up in my junk folder) essentially letting me know that someone was trying to access my Mitro account from an unrecognized device/browser. All I had to do was click the link in that e-mail (which went to my junk folder and didn't show up for a few hours anyway) and now I can access Mitro from both Chrome and Firefox, even with the newer versions of FF. 

Hope this helped!
